app/src/components/fields/CbSelectField.tsx

```diff
@@ -50,12 +50,16 @@ const CbSelectField: React.FC<ICbSelectField> = (props) => {
   useEffect(() => {
     if (hasLoaded) {
       // Only refresh when the query or param changes
-      isMounted() && refresh({ route, param, query });
+      refresh({ route, param, query });
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [query, param]);
 
   const isValueInRange = useMemo(() => {
```

Comment:
It seems like this function is doing more than it should be, based on the name.

It looks like it should just a basic 'is' function (is value in range or not), but it is also calling 'setFieldValue'. Possibly that should be pulled out, so that this function can just be a pure 'is' function? And the setFieldvalue calls should be made separately after this call is made

Reply:
You wouldn't need the isMounted check either then, as this function wouldn't have any set state calls. Would be simpler overal.